src/ConfigDefinition.php

```diff
@@ -15,12 +15,15 @@ protected function getParametersDefinition(): ArrayNodeDefinition
         $parametersNode
             ->children()
                 ->enumNode('compression_type')
-                    ->values(['auto', 'zip', 'gzip', 'snappy'])
+                    ->values(['auto', 'zip', 'gzip', 'snappy', 'zlib'])
                     ->defaultValue('auto')
                 ->end()
                 ->booleanNode('graceful')
                     ->defaultValue(false)
                 ->end()
+                ->integerNode('zlib_window_size')
+                    ->defaultValue(-15)
```

Comment:
podle dokumentace je default +15 https://docs.python.org/3/library/zlib.html, dal bych to tam (= je to standard)